.github/actions/checkout-submodules-and-bootstrap/action.yaml

```diff
@@ -28,6 +28,14 @@ runs:
       uses: ./.github/actions/bootstrap
       with:
         platform: ${{ inputs.platform }}
+    - name: Remove CIPD cache directory
+      # If there is a bootstrap-cache miss, and bootstrap installs CIPD itself,
+      # there is roughly 1.4GB that remains in this cache that is useless from
+      # this point onwards.
+      shell: bash
+      run: |
+        du -sh $HOME/.cipd-cache-dir/
+        rm -rf $HOME/.cipd-cache-dir/
```

Comment:
Leaving blocking comment since I discovered if the directory doesn't exist this can cause the job to fail, which we expect would happen often as we expect more cache-hits then failures